tests/chaos/oadp/conftest.py

```
@@ -157,10 +161,214 @@ def backup_with_pod_deletion_orchestration(
+    for _ in TimeoutSampler(
+        wait_timeout=TIMEOUT_3MIN,
+        sleep=POLL_INTERVAL,
+        func=all_vms_deleted,
+    ):
+        if all_vms_deleted():
+            break
+
+    LOGGER.info(f"All VMs in namespace {namespace_name} deleted")
+
+
+@pytest.fixture()
```

Ruff (0.15.0)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)